config.json

```diff
@@ -36,6 +36,12 @@
       "values": ["win7", "osx.10.10", "ubuntu.14.04", "ubuntu.16.04", "etc-other-rid-based-os-names"],
       "defaultValue": ""
     },
+    "HostOS": {
+      "description": "The RID used for restoring native OS dependent assets (i.e. RID specific binaries)",
```

Comment:
It's still not clear to me from these two descriptions how this differs from RuntimeOS, we need to make that crisp.